packages/ui-kit/src/design-system.js

```diff
@@ -124,6 +124,7 @@ export const tokens = {
 
 export const dimensions = {
   heights: {
+    onePixel: '1px',
```

Comment:
Let's add a comment mentioning the reasons for this special case.

PS: this value is not really used as a height. I think I would prefer to inline it in the code instead of using a variable, because we don't really want to encourage the usage of 1px spacing.

Reply:
Allright 👍